erpnext_com/www/docs/user/manual/en/setting-up/stock-reconciliation.md

```diff
@@ -180,6 +191,34 @@ As you can see, the Balance Qty as on 10th January got updated from 100 to 120.
 	</iframe>
 </div>
 
+####4.2 For Serialized Items
+
+For item ITEM-00225 has the 6 serial nos HJF00020, HJF00021, HJF00022, HJF00023, HJF00024, HJF00025 with valuation rate as 496.67 per serial no. At the end of the year user has came to know that they have only 3 serial nos against that item with valuation rate 530. So to remove the old serial nos HJF00020, HJF00021, HJF00022, HJF00023, HJF00024, HJF00025 and add the new serial nos with new valuation rate stock reconciliation can be use as follows
+
+Select the item ITEM-00225 in the stock reconciliation, on selection of the item system will auto pulled the existing serials nos. Then set qty as 3, valuation rate as 530 and serial no as HJF00026, HJF00027, HJF00028.
+
+
+<img class="screenshot" alt="Stock Reconciliation" src="{{docs_base_url}}/assets/img/setup/stock-recon-for-serialized.png">
+
+Before reconciliation, the valuation rate was 530 and available qty was 6, so the total stock value was 3,180. After reconciliation the valuation rate has changed to 620 and available qty changed to 3, so the new stock value becomses the 1,860. To adjust the stock value in the accounting, system has credited extra amount 3,180 - 1,860 = 1,320 to warehouse's account and debited to stock adjustment account. The gl entries for the above entry is as follows
+
+<img class="screenshot" alt="Stock Reconciliation" src="{{docs_base_url}}/assets/img/setup/gl_entry_for_serialized_items.png">
+
+The stock balance after submission of the stock reconciliation
+<img class="screenshot" alt="Stock Reconciliation" src="{{docs_base_url}}/assets/img/setup/stock_balance_after_stock_reco_submission.png">
+
+The general ledger for the warehouse account Nagpur after submission of the stock reconciliation
+<img class="screenshot" alt="Stock Reconciliation" src="{{docs_base_url}}/assets/img/setup/general_ledger_after_stock_reco_submission.png">
+
+####4.3 For Batch Items
+
+Stock reconciliation for batch items will be use to add new batch or to update the quantity of the existing batch. For example the batch JHGJH00003 has the current quantity as 60 but user wants to make it 100 then using stock reconciliation user can update the batch quantity.
```

Comment:
Stock reconciliation for batch items will be used to add a new batch or to update the quantity of the existing batch. For example, the batch JHGJH00003 has the current quantity as 60 but if the user wants to make it 100 then by using stock reconciliation, user can update the batch quantity.